### DISCUSSION TITLE
How do I modify the default values of a profile rather than individual users modify the default settings?

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4848] [Plugins]
How do I modify the default values of a profile rather than individual users modify the default settings?

Where do I change the default settings in the kimai2_user_preferences table?

**Comment by kevinpapst** [comment]
System > Settings has a few default values which only apply to new user accounts.
Existing user accounts have to be changed individually.

**Reply by seleondevel** [reply]
![스크린샷 2024-05-10 135935](https://github.com/kimai/kimai/assets/165639563/c0621fe7-7aac-4c3e-a914-c4ffecf91249)
The initial view settings after login are not in the system settings. Where can I default on this?

**Reply by seleondevel** [reply]
![스크린샷 2024-05-10 121829](https://github.com/kimai/kimai/assets/165639563/64b8c90d-5dfa-411c-ac43-04f636232c05)

And the default setting for the Table columns on the Customize display is selected as Team and Actions, so I want to take out Team, how can I change the default setting?

**Reply by seleondevel** [reply]
kimai\src\Controller\HomepageController.php

Line 29

 //public const DEFAULT_ROUTE = 'timesheet';
 public const DEFAULT_ROUTE = 'dashboard';
    
I modified the code, updated kimai2_user_presentSET value = 'dashboard' where name = 'login_initial_view'; query executed.

**Reply by seleondevel** [reply]
I would like to exclude team:true from kimai2_bookmarks content as the default setting.

I can't see the code setting columns as default in kimai2_bookmarks, what should I do?

**Reply by kevinpapst** [reply]
You have to change the default search options for that: https://github.com/kimai/kimai/tree/main/src/Form/Toolbar

**Reply by kevinpapst** [reply]
And btw: I cannot give you support for all these customizings. 
What exactly are you doing? Your company or customer will never be able to properly update their system.
